pixi.toml

```diff
@@ -9,7 +9,7 @@ platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
 
 [dependencies]
 python = ">=3.9"
-polars = ">=0.14.24,<0.21"
+polars = ">=0.14.24,<2"
```

Comment:
I think we should pin to <1.1

Reply:
shouldn't the next breaking change be polars 2? since we are not touching polars internals but only the public api, this should be fine imo. but if you have strong opinions on this, we can also pin to <1.1

Reply:
They might add unsupported syntax without bumping the major version. Let's keep a rather tight pin to 1.1 or 1.2 here